content/news/playtest-20170826.md

```diff
@@ -0,0 +1,16 @@
+A month after the previous playtest, we're pleased to announce the new release candidate, playtest-20170826.
+
+In addition to all the new features and improvements the previous playtest brought along, in this new playtest we fixed over 30 issues that were identified through player and modder feedback. Among the more noticable ones were the movement 'stuttering' that ground units were showing, as well as some issues with the new command bar and some other UI hiccups.
```

Comment:
It would be good to change "the previous playtest" to a link to the last news post.